src/App.js

```diff
@@ -1,24 +1,70 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './css/index.css';
+import Form from './Components/Form';
+import Lyrics from './Components/Lyrics';
 
 function App() {
+  const [song, setSong] = useState();
+  const [band, setBand] = useState();
+  const [lyrics, setLyrics] = useState();
+  const [loading, setLoading] = useState();
+
+  const handleSong = (e) => {
+      setSong(normalize(e.target.value));
+  }
+
+  const handleBand = (e) => {
+      setBand(normalize(e.target.value));
+  }
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    setLyrics();
+    setLoading(spinner);
+
+    try {
+      const lyricsResponse = await findLyrics(band, song);
+      const data = await lyricsResponse.json();
+      if (data.lyrics) {
+        setLyrics(data.lyrics);
+      }
```

Comment:
Are you using some auto-formatter? You could try [prettier](https://prettier.io) for javascript code.

Reply:
I didn't use it in this project but it's fixed now, I run JS prettier.